src/ServiceBouncer/MainForm.cs

```diff
@@ -6,7 +6,10 @@
 using System.Threading;
 using System.Threading.Tasks;
 using System.Windows.Forms;
+using CredMan = CredentialManagement;
```

Comment:
Can we not alias the types like this please.

Reply:
Okay. Then I will remove this using and call the types by the full namespace path. (There wer conflicts)